index.html

```diff
@@ -21,41 +21,22 @@
         </div>

```

Comment:
Could you activate gh-pages on your repo and allow me to see the live version of the changes that are made here?

Reply:
Okay